nntrainer/graph/network_graph.cpp

```diff
@@ -752,9 +753,8 @@ NetworkGraph::finalizeContext(const std::shared_ptr<LayerNode> &lnode,
   return outputs;
 }
 
-int NetworkGraph::initialize(
-  const std::vector<std::string> &model_input_names,
-  const std::vector<std::string> &model_label_names) {
+int NetworkGraph::initialize(const std::vector<Connection> &model_input_names,
+                             const std::vector<Connection> &model_label_names) {
```

Comment:
/s/model_label_names/model_output_names/ ?

Reply:
Current semantics is label, note that label and output is and will be disambiguated.